nuagekubemon/policy/resourcemanager.go

```diff
@@ -136,8 +137,70 @@ func (rm *ResourceManager) GetPolicyGroupsForPod(podName string, podNs string) (
 	return &pgList, nil
 }
 
+const (
+	zoneAnnotationAdd    = "ZoneAnnotationAdd"
+	zoneAnnotationRemove = "ZoneAnnotationRemove"
+)
+
+func (rm *ResourceManager) createNuagePolicyUpdateForZoneAnnotation(namespace string, op string) error {
```

Comment:
Since we are updating the annotation ingress/egress ACL template by adding or removing enteries,